src/_pytest/fixtures.py

```diff
@@ -426,7 +426,8 @@ def module(self):
     @scopeproperty()
     def fspath(self) -> py.path.local:
         """ the file system path of the test module which collected this test. """
-        return self._pyfuncitem.fspath
+        # TODO: Remove ignore once _pyfuncitem is properly typed.
+        return self._pyfuncitem.fspath  # type: ignore
```

Comment:
Use the error code here?  (see `--show-error-codes`, could be added to the config probably)  (with flake-ignore then)

Reply:
We should probably do them all at once, maybe on the next mypy bump.